site/docs/components/landing/NavigationList.tsx

```diff
@@ -81,6 +81,28 @@ const swapSvg = (
   </svg>
 );
 
+const transactionSvg = (
```

Comment:
This page is getting pretty long, let's get all those SVGs into `docs/components/svg/`.